spec/models/wishlist_spec.rb

```diff
@@ -3,4 +3,10 @@
 RSpec.describe Wishlist, type: :model do
   it { is_expected.to belong_to :user }
   it { is_expected.to have_many :items }
+
+  it "creates a token for every new wishlist created" do
+    # new_wishlist could be changed for let: when more tests are present
+    new_wishlist = Wishlist.create(name: "Name", description: "Description")
```

Comment:
I also had future merges in mind while leaving it like this. But OK, I will change it.